core/bcast/bcast.go

```diff
@@ -93,6 +95,24 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty,
 	case core.DutyRandao:
 		// Randao is an internal duty, not broadcasted to beacon chain
 		return nil
+	case core.DutyVoluntaryExit:
+		// JSON decoding from the previous component
+		ve := new(eth2p0.SignedVoluntaryExit)
+		err := json.Unmarshal(aggData.Data, ve)
+		if err != nil {
+			return errors.Wrap(err, "json decoding voluntary exit")
+		}
+
+		err = b.eth2Cl.SubmitVoluntaryExit(ctx, ve)
+
+		if err == nil {
+			log.Info(ctx, "Voluntary exit successfully submitted to beacon node",
+				z.U64("epoch", uint64(ve.Message.Epoch)),
+				z.U64("validatorIndex", uint64(ve.Message.ValidatorIndex)),
```

Comment:
please use snake case for json logging fields (see guidelines)

Reply:
I would also add the pubkey here: `z.Any("pubkey", pubkey)`

Reply:
Done